src/shared/services/toast.service.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@angular/core';
+import { ToastrService } from 'ngx-toastr';
+import { TranslateService } from '@ngx-translate/core';
+
+@Injectable()
+export class ToastService {
+  private readonly toastConfigObject = {
+    positionClass: 'toast-bottom-center',
+  };
+
+  constructor(private toastr: ToastrService, private translate: TranslateService) {}
```

Comment:
Let's make a simple service here.
1. Remove the translate service here.
2. Move it to places where you're passing the messages.
3. Passed messages should be already translated.
4. Create 4 methods in this service - show[Success/Info/Error/Warting]Toas(message: string)